### PR TITLE
Contribute Aiven Python template.

### PR DESCRIPTION
**Pull request by dewan-ahmed**
This PR adds a Python template for Aiven. Currently, there is one Aiven template available for Go.

In addition, the PR updates the Aiven for Apache Kafka version to 3.2. The version 2.7 in the current example is outdated. 

The PR also updates some of the variable values for consistency and ease of understanding.

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
A note that `project="<YOUR_AIVEN_PROJECT_NAME>"` needs to be updated with an actual Aiven project name before running a test.

**Comment by cnunciato** [comment]
Hi @dewan-ahmed -- thanks for these contributions! We'll take a look today.

**Comment by dewan-ahmed** [comment]
Thanks @cnunciato

File: aiven-python/Pulumi.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: python
+template:
+  description: A minimal Aiven Python Pulumi program
+  config:
+    aiven:apiToken:
+      description: The token that allows you access to your Aiven account
+      secret: true
```
Comment (cnunciato) [inline aiven-python/Pulumi.yaml:9]: Same comment as https://github.com/pulumi/templates/pull/480#discussion_r1067559807.

File: aiven-python/__main__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import pulumi
+import pulumi_aiven as aiven
+
+kafka1 = aiven.Kafka("kafka1",
```
Comment (cnunciato) [inline aiven-python/__main__.py:4]: Similarly to https://github.com/pulumi/templates/pull/480#discussion_r1067575686:

```suggestion
# Create a Kafka service.
kafka = aiven.Kafka("kafka",
```

File: aiven-python/__main__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    project="<YOUR_AIVEN_PROJECT_NAME>",
+    cloud_name="google-europe-west1",
+    plan="business-4",
+    service_name="kafka-gcp-eu",
```
Comment (cnunciato) [inline aiven-python/__main__.py:8]: Similar to https://github.com/pulumi/templates/pull/480#discussion_r1067569171, I'd suggest we make these configurable.

File: aiven-python/__main__.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        kafka=aiven.KafkaKafkaUserConfigKafkaArgs(
+            group_max_session_timeout_ms="70000",
+            log_retention_bytes="1000000000",
+        ),
+        public_access=aiven.KafkaKafkaUserConfigPublicAccessArgs(
+            kafka_rest="true",
+            kafka_connect="true",
+        ),
+    ))
+
```
Comment (cnunciato) [inline aiven-python/__main__.py:25]: Similar to https://github.com/pulumi/templates/pull/480#discussion_r1067578728, we should probably export the most relevant results:

```suggestion
# Export the service host and port.
pulumi.export("serviceHost", service.host)
pulumi.export("servicePort", service.port)
```

File: aiven-go/main.go [hunk 5]
```diff
@@ -8,12 +8,12 @@ import (
 func main() {
 	pulumi.Run(func(ctx *pulumi.Context) error {
 		kafka, err := aiven.NewKafka(ctx, "aiven-kafka", &aiven.KafkaArgs{
```
Comment (cnunciato) [inline aiven-go/main.go:10]: Similar comments here as those provided on `aiven-python` and `aiven-typescript`.

**Review by cnunciato — COMMENTED** [review]
Again, thanks for the contribution! Some initial feedback here as well -- all pretty much aligned with what I submitted on https://github.com/pulumi/templates/pull/480.

**Comment by dewan-ahmed** [comment]
Thanks for the review @cnunciato 

I added a few commits to https://github.com/pulumi/templates/pull/480 and asked some questions. Once I hear your suggestions there, I'll commit changes on this PR accordingly.

**Comment by cnunciato** [comment]
@dewan-ahmed #480 is merged! 🎉  If you could apply those same changes here, we'll get this approved and merged for you as well.

**Comment by dewan-ahmed** [comment]
Thanks @cnunciato, I added some updates via https://github.com/pulumi/templates/pull/479/commits/252be0cc1a10bfa6a00bc11a81305107db720830

My Python skills are outdated so please check if anything needs fix. I just noticed that I forgot to add the default values for plan, cloud and service_name in the TypeScript example. I'll create another PR to update the default config values for TS and GoLang examples.

Also, I renamed some of the config names to avoid confusing them with the variable names. Waiting to hear back on your review and can make changes accordingly 🙌🏽

**Review by cnunciato — APPROVED** [review]
@dewan-ahmed This is great! Thanks for all the revisions. I'll probably circle back and make those Go template settings configurable as well, to align with the TS and Python templates, so I may tag you on that review, but that needn't prevent this one from merging.

Thanks again for the contribution! 🙌

**Comment by dewan-ahmed** [comment]
Thanks @cnunciato ; much appreciated! It has been a pleasant contributing experience.

**Pull request merged**